outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "webapp_url" {
+    value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+output "cd_webhook" {
+    value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
This leaks your deployment password because it will appear in CI/CD logs after terraform apply completes.

You should add `sensitive = true` to this output.

Similarly you add `sensitive = true` to any sensitive "variables" to prevent them appearing in the logs. (Which you have done already). You may not see variables appear in logs currently but there are times when they would.